include/swift/AST/DiagnosticsSema.def

```diff
@@ -925,6 +925,12 @@ NOTE(unwrap_with_default_value,none,
 NOTE(unwrap_with_force_value,none,
      "force-unwrap using '!' to abort execution if the optional value contains "
      "'nil'", ())
+NOTE(unwrap_iuo_initializer,none,
+     "initializing from an implicitly unwrapped optional is inferred to be "
+     "optional in Swift 4", ())
```

Comment:
This was true in Swift 3 in general.

Reply:
It's been longer than I remembered! Will fix.